src/GraphQL/ParseGraphQLServer.js

```diff
@@ -39,11 +45,20 @@ class ParseGraphQLServer {
           auth,
         }),
         maskedErrors: false,
-        multipart: {
-          fileSize: this._transformMaxUploadSizeToBytes(
-            this.parseServer.config.maxUploadSize || '20mb'
-          ),
-        },
+        // Needed to ensure formDataLimits since it seems to not working
+        // this is a temporary fix until the issue is resolved
+        // we need to ask graphql-yoga team
+        plugins: [
```

Comment:
This is the recommended way of configuring FormData limits;
https://the-guild.dev/graphql/yoga-server/docs/features/file-uploads#configuring-multipart-request-processing-only-for-nodejs

Reply:
We're testing it here;
Test -> https://github.com/dotansimha/graphql-yoga/blob/main/packages/graphql-yoga/__integration-tests__/incremental-delivery.spec.ts#L302
Configuration -> https://github.com/dotansimha/graphql-yoga/blob/main/packages/graphql-yoga/__integration-tests__/incremental-delivery.spec.ts#L192

Reply:
i tried without the plugin and just the config on the Yoga docs, but i encountered a bug. The format data options seems to be lost during the request processing so the multipart limit don't work. This is why i needed to add the plugin to enforce the option.

Reply:
Like I said above, it is not part of Fetch API spec. It is VERY specific to our implementation. I'd strongly recommend to avoid this kind of testing. If something is not working as expected, you can help us to reproduce it on our repo.

Reply:
I understand that the issue is quite specific, @ardatan. I'm not particularly fond of relying on this workaround myself. I invested a considerable amount of time investigating into this issue. I've managed to identify a reproducible case within the Parse Server test suite. However, I'll not be able to dedicate time to reproduce the bug on Yoga repo.

Given that Parse is designed to be a user-friendly Backend as a Service (BaaS), it's crucial to ensure the seamless functionality of all its features. Therefore, we conduct integration tests with our dependencies to guarantee the production readiness and stability of the software.

This is precisely why I reached out to the Guild for assistance. It would be greatly appreciated if you could investigate the bugs within our test suite, as I suspect they might all be linked to the 'fetch ponyfill' library also maintenained by the guild team.

If you find that investigating into Yoga-related bugs within the Parse Server test suite isn't within your scope of interest, please don't hesitate to let me know. I completely understand and respect that. Additionally, I intend to experiment with the new Apollo Server v4. Perhaps within the context of the Parse Server, we'll encounter fewer issues.

It's not an ultimatum; perhaps Yoga is well-suited for certain use cases, but in the context of Parse Server, there appear to be some issues.


I also admire all the work that the guild has done on tools like GraphQL codegen :)